themes/custom/az_barrio/templates/navigation/pager.html.twig

```diff
@@ -0,0 +1,96 @@
+{#
+/**
+ * @file
+ * Theme override to display a pager.
+ *
+ * Available variables:
+ * - items: List of pager items.
+ *   The list is keyed by the following elements:
+ *   - first: Item for the first page; not present on the first page of results.
+ *   - previous: Item for the previous page; not present on the first page
+ *     of results.
+ *   - next: Item for the next page; not present on the last page of results.
+ *   - last: Item for the last page; not present on the last page of results.
+ *   - pages: List of pages, keyed by page number.
+ *   Sub-sub elements:
+ *   items.first, items.previous, items.next, items.last, and each item inside
+ *   items.pages contain the following elements:
+ *   - href: URL with appropriate query parameters for the item.
+ *   - attributes: A keyed list of HTML attributes for the item.
+ *   - text: The visible text used for the item link, such as "‹ Previous"
+ *     or "Next ›".
+ * - current: The page number of the current page.
+ * - ellipses: If there are more pages than the quantity allows, then an
+ *   ellipsis before or after the listed pages may be present.
+ *   - previous: Present if the currently visible list of pages does not start
+ *     at the first page.
+ *   - next: Present if the visible list of pages ends before the last page.
+ *
+ * @see template_preprocess_pager()
+ */
+#}
+{% if items %}
+  <nav aria-label="{{ heading_id }}">
+    <h3 id="{{ heading_id }}" class="sr-only">{{ 'Pagination'|t }}</h3>
```

Comment:
```
<h3 id="{{ heading_id }}" class="sr-only">{{ 'Pagination'|t }}</h3>
```
This heading level (was H4) is the only thing changed from the [original file](https://git.drupalcode.org/project/bootstrap_barrio/-/blob/8.x-4.x/templates/navigation/pager.html.twig?ref_type=heads).